File: Programmers/연습문제/Level 1/x만큼 간격이 있는 n개의 숫자/Solution.swift
import Foundation

func solution(_ x:Int, _ n:Int) -> [Int64] {
    var answer: [Int64] = []
    
    for i in stride(from: 0, to: n, by: 1) {
        answer.append(Int64(x + x*i))
    }
    return answer
}

File: Programmers/연습문제/Level 1/x만큼 간격이 있는 n개의 숫자/Solution.js
function solution(x, n) {
    var answer = getAnswer(x, n);
    return answer;
}

function getAnswer(x, n) {
    const answer = [];
    let num = x;
    while (answer.length < n) {
        answer.push(num);
        num += x;
    }
    return answer;
}